src/components/ordersPage.vue

```diff
@@ -2,46 +2,6 @@
   <div>
     <h2 class="text-center">Orders</h2>
 
```

Comment:
Would we want to keep this method on this page, or at least maybe leave a button to re-route to the client page, or is it not necessary for admin access?

Reply:
I took it out here because it's not really necessary to have it on the orders page, since it directly relates to the clients even if it's for orders.